shippable.yml

```diff
@@ -21,6 +21,7 @@ matrix:
     - env: T=units/3.7
     - env: T=units/3.8
     - env: T=units/3.9
+    - env: T=units/3.10
```

Comment:
We have moved our CI to Azure Pipelines. Changes to test matrix should be made in `.azure-pipelines/azure-pipelines.yml`.